File: Baekjoon_Online_Judge/NO_18795/18795.cpp
#include <iostream>
using namespace std;
/*
 -------------------------------------- 
 | Created by happiness96             | 
 | Year 2020                          | 
 | Month 11                           | 
 | Day 24                             | 
 | 18795 이동하기 3                    | 
 -------------------------------------- 
*/

int main(){
    ios::sync_with_stdio(0);
    cin.tie(0);

    long long N, M, total = 0;

    cin >> N >> M;

    for(int i = 0; i < N + M; i ++){
        long long k;

        cin >> k;

        total += k;
    }

    cout << total;
    return 0;
}

File: Baekjoon_Online_Judge/NO_18795/18795.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# -------------------------------------- #
# | Created by happiness96             | #
# | Year 2020                          | #
# | Month 11                           | #
# | Day 24                             | #
# | 18795 이동하기 3                   | #
# -------------------------------------- #


def run():
    N, M = map(int, r_input().split())

    print(sum(map(int, r_input().split())) + sum(map(int, r_input().split())))


if __name__ == "__main__":
    run()
